src/app/main/service/localstorage/local-storage.service.ts

```diff
@@ -27,6 +27,18 @@ export class LocalStorageService {
   ubsRegBehaviourSubject: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(this.getUbsRegistration());
   ubsRedirectionBehaviourSubject: BehaviorSubject<number> = new BehaviorSubject<number>(this.getOrderIdToRedirect());
 
+  public setIsChatOpen(value: boolean): void {
+    localStorage.setItem(this.CHAT_OPEN, JSON.stringify(value));
+  }
+
+  public getIsChatOpen(): boolean | null {
+    return JSON.parse(localStorage.getItem(this.CHAT_OPEN));
```

Comment:
Consider returning a `boolean` default value (e.g., false) to avoid ambiguity in the return type.
you can use `=== null ? false `